src/main/java/com/artipie/asto/fs/FileStorage.java

```diff
@@ -47,6 +47,11 @@
 /**
  * Simple storage, in files.
  *
+ * <p>
+ * This class must be constructed only via vertx instance, otherwise we can get file system from
+ * another vertx instance and it will cause performance issues.
```

Comment:
Does the passing of another vert.x instance would not cause the mentioned "performance issues"?